### DISCUSSION TITLE
传火吗？绳匠

### DISCUSSION THREAD

**Discussion by zifeiyu1111** [share121/inter-knot#1454] [其他]
![v2-19b7ab946c5ad6eb9365b5687c7cc8d2_1440w](https://github.com/user-attachments/assets/512edc05-6d62-4b32-9ea7-3f0c88fa5b0c)

**Comment by kallen1314** [comment]
手残，没资格传

**Comment by strt13** [comment]
小高我比你更胜一筹

**Comment by twilight-27** [comment]
啥意思

**Comment by CaptainHarryChen** [comment]
Yes, indeed!
